Dockerfile

```diff
@@ -19,7 +13,7 @@ COPY go.sum .
 RUN go mod download && go mod verify
 
 COPY . .
-RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -ldflags="-w -s" -o /opt/go-didx509-toolkit/didx509-toolkit
```

Comment:
The binary size increases by 50% if these flags are removed.
Fine either way, but the build command in the makefile should probably be the same as this

Reply:
Ah I assumed they were CGO related. I'll revert this.